### DISCUSSION TITLE
Jetbrains Fleets support

### DISCUSSION THREAD

**Discussion by kkaansrky** [izhangzhihao/intellij-rainbow-brackets#2688] [Show and tell]
I got so used to this plugin when I was writing Android. I am currently using fleets while writing KMM. Will there be support for Fleets?

**Comment by izhangzhihao** [comment]
The toolchain for building Fleet plugins is closed source and has not been publicly released. Without these supports, I am currently unable to provide support for Fleet. However, I have applied to join the public testing program, but there has been no progress as of now.

**Reply by kkaansrky** [reply]
Then I impatiently await news of support from you. Thank you for the quick reply. 🚀 💯

**Comment by izhangzhihao** [comment]
Hi, basic features are testing in my Fleet IDE, I'm still trying to get the first version of Fleet published.
I cannot share the plugin build snapshot until it's approved by JetBrains Marketplace.
If you think this plugin can help, consider donate at https://opencollective.com/intellij-rainbow-brackets/projects/fleet-support

Here are some screenshots:

<img width="867" alt="image" src="https://github.com/izhangzhihao/intellij-rainbow-brackets/assets/12044174/2ba7f3ed-f1f9-456c-837a-115f3ca94189">


<img width="548" alt="image" src="https://github.com/izhangzhihao/intellij-rainbow-brackets/assets/12044174/a1019b99-271b-4851-81e7-39cebb42b01f">


<img width="815" alt="image" src="https://github.com/izhangzhihao/intellij-rainbow-brackets/assets/12044174/28cb3b65-a595-46f7-9e46-52372dd2a888">


<img width="930" alt="image" src="https://github.com/izhangzhihao/intellij-rainbow-brackets/assets/12044174/d86c0817-aeff-4e7e-8351-459102c344ed">

**Comment by izhangzhihao** [comment]
Pls follow this to install Rainbow Brackets into your Fleet IDE: https://github.com/izhangzhihao/intellij-rainbow-brackets/blob/2022.3/Fleet.md

**Reply by azeek** [reply]
Currently, it is published as a separate plugin. If a paid official version is released in the future, will existing JetBrains IDE plugin paid license users also need to purchase a separate paid license?

**Reply by izhangzhihao** [reply]
Not sure now, since the Intellij based plugin and Fleet based plugin have very difference structure, makes it very hard to share the same code and build script. And JetBrains still not share any information about paid plugins for the Fleet yet.

**Comment by izhangzhihao** [comment]
Closed since [Fleet is dead](https://blog.jetbrains.com/fleet/2025/12/the-future-of-fleet/) now.
